scripts/compare_yamls.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"log"
+	"os"
+	"strings"
+
+	"gopkg.in/yaml.v3"
+)
+
+func main() {
+	exampleConfigFile, err := os.ReadFile("config/example_config.yaml")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	newConfigFile, err := os.ReadFile("config/dumped_config.yaml")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// unmarshal ymal
+	exampleConfig := make(map[string]interface{})
+	if err := yaml.Unmarshal(exampleConfigFile, &exampleConfig); err != nil {
+		log.Fatal(err)
+	}
+	exampleConfig = convertToLowercase(exampleConfig)
```

Comment:
What happens if example config is not converted to lowercase?
I think that should not be necessary, not?

Reply:
Viper is case-insensitive. It treats all values as lowercase, and therefore dumps the config in all lowercase (try to run `go run ./cmd/main/arc -dump_config=./dumped_config.yaml` and look at the keys).

Maps in go, on the other hand, are case-sensitive. So, if we use cammelCase in `example_config.go`, it will be reflected in the unmarshalled map. Therefore, the 2 configs will not be equal in terms of maps in go:
```yaml
logLevel: INFO
loglevel: INFO
```
but they will be perfectly equal in viper, which is what we want to check here.

That's why I'm converting the example config to lowercase :)